### PR TITLE
Updates based on feedback for document uploader

### PR DESCRIPTION
**Pull request by bseeger**
In /src/test, this moves "utils" -> "testutils" to make a space to do unit tests for main "utils".

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ ethnicity-selection.option1=Hispanic or Latino
 ethnicity-selection.option2=Not Hispanic or Latino
 
 race-selection.title=Race Selection
-race-selection.header=What races do you ({0}) identify with?
+race-selection.header=What race`s do you ({0}) identify with?
```
Comment (bseeger) [inline src/main/resources/messages.properties:1021]: Changing this back. Not sure what happened here, but it's def. not right.

Comment (vrajmohan) [reply src/main/resources/messages.properties]: Hmm...if we expect only one to be chosen, it should be "race", else it should be "races". "race's" is wrong.

File: src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class TestUserFileUtilities {
+    @CsvSource(delimiter=';', value = {
+        // passed in value; expected value
+        ".jpg, .jpeg, .csv,.pdf;.jpg, .jpeg, .csv, and .pdf",
```
Comment (vrajmohan) [inline src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java:11]: Feel silly mentioning it, but as we are doing joins and Oxford commas, I would add cases for 1 file type and 2 file types.

**Review by vrajmohan — APPROVED** [review]
Minor comment

File: src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java (hunk 2 above)
Comment (bseeger) [reply src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java:11]: Oh, that's a very good point.

Comment (bseeger) [reply src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java]: I added code to address this.  I think it was good to tighten this up some.

**Review by vrajmohan — APPROVED** [review]
Looks great

**Pull request merged**